mops.toml

```
@@ -1,8 +1,7 @@
 [dependencies]
-base = "0.12.1"
+base = "0.13.4"
```

Are we able to use the latest base library version?

```suggestion
base = "0.14.1
```

---

I had that, lamentably it is not compatible as a primitive is missing. I'll update when `dfx` 0.25 is out.